README.md

```diff
@@ -51,6 +51,8 @@ When working and testing locally, you can also install the `fsd_utils` package f
     pip uninstall -y funding-service-design-utils
     pip install /path/to/your/working/directory/funding-service-design/utils
 
+Note: When testing locally using the docker runner, docker might use the cached version of fsd_utils. to avoid this and pick up your intended changes, run `docker compose build <service_name> --no-cache` first before running `docker compose up` or just running `docker compose up --build` on its own.
+
```

Comment:
love the extra docs ❇️  might this hint be more helpful in the docker runner repo?

Reply:
ahh good shout! I'll add this note on the docker runner readme too 👍🏾